run_operations.py

```diff
@@ -1,394 +1,293 @@
-#!/usr/bin/env python3
-
+#!/usr/bin/env python3 
 import os
 import sys
 import subprocess
 import argparse
-from datetime import datetime
+from datetime import date, datetime
 import shutil
 import time
 import glob
 
-import subprocess
-################### directory initiation for Josh to review (and remove comment)
-##################
-
 import generate_template_files as gt
+from rsmas_logging import RsmasLogger, loglevel
+from dataset_template import Template
+import _process_utilities as putils
 
-import logging
+OPERATIONS_DIRECTORY = os.getenv('OPERATIONS')
+STORED_DATE_FILE = OPERATIONS_DIRECTORY + "/stored_date.date"
+TEMPLATE_DIRECTORY = OPERATIONS_DIRECTORY + "/TEMPLATES"
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%S.%f"
 
```

Comment:
Move "ERRORS_DIRECTORY", "LOGS_DIRECTORY" and delete lower case variables in `initiate`